lib/fpm/package.rb

```diff
@@ -205,7 +205,8 @@ def convert(klass)
     ivars.each do |ivar|
       #logger.debug("Copying ivar", :ivar => ivar, :value => instance_variable_get(ivar),
                     #:from => self.type, :to => pkg.type)
-      pkg.instance_variable_set(ivar, instance_variable_get(ivar))
+      ival = instance_variable_get(ivar)
+      pkg.instance_variable_set(ivar, ival != '' ? ival : nil )
```

Comment:
You can use [`#empty?`](https://ruby-doc.org/core-2.2.0/String.html#method-i-empty-3F) and flip the logic around a little. That's about it as far as plain Ruby goes.

```ruby
ival.empty? ? nil : ival
```